include/functions_routing.inc.php

```diff
@@ -428,66 +430,39 @@ function serveArchives() {
         default:
             $gday = 1;
 
-            if ($week) {
```

Comment:
I don't understand this fully yet, but doesn't it break a feature or misses some logic check now?!

Reply:
I think this code never worked, $week would never be initialized. This must have been broken before.

Reply:
I think in earlier codes (maybe before the change of proper routing was introcued) the $week parameter was possible to come from the URL directly `/archives/W43.html`...

Reply:
I'd assume so as well, but I grepped for $week and nothing came up outside of .tpl files. Where was this assigned? Do you know enough about how this was supposed to work to restore it? I know nothing about the archive - but if we can we should of course fix in (in a separate PR).